pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/generator/DirectiveGenerator/index.jelly

```diff
@@ -43,7 +43,7 @@
                 </f:block>
                 <f:section title="${%Directives}"/>
                 <!-- Similar to f:dropdownDescriptorSelector, but adds fallback content to block, and JENKINS-25130 adds per-selection help: -->
-                <j:set var="item" value="${it.getItem(request)}"/>
+                <j:set var="item" value="${it.getItem(request2)}"/>
```

Comment:
This seems like a fix for a recent regression due to picking up a newer `workflow-cps` dep, right? So this much could be filed separately.

Reply:
I merged this separately in #777.